nav/support/correlate.py

```
@@ -1,701 +1,553 @@
+# ==============================================================
+
+def evaluate_candidate(*,
+                       image_pad: NDArrayFloatType,
+                       model_pad: NDArrayFloatType,
+                       mask_pad: NDArrayBoolType,
+                       corr: NDArrayFloatType,
+                       rc: tuple[int, int],
+                       upsample_factor: int,
+                       model_shape: tuple[int, int],
+                       image_shape: tuple[int, int],
+                       logger: PdsLogger,
```

Ruff (0.14.4)

Unused function argument: `logger`

(ARG001)